core/docker/compose.sh

```diff
@@ -2,8 +2,16 @@
 
 composeFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
+if [ "$PROD" == "true" ]; then
+    devComposeParam=""
```

Comment:
Perhaps add a printf to say it's in PROD mode?
```suggestion
    printf "\nRunning core package in PROD mode\n"
    devComposeParam=""
```